enterprise-suite/scripts/run-e2e-tests.sh

```diff
@@ -45,7 +45,7 @@ if [[ "$?" != "0" ]]; then
     exit 1
 fi
 
-if [[ -z "$subset" ]]; then
+if [[ "$subset" == "all" ]]; then
```

Comment:
Why not make the default to run all of them and require a subset parameter for just 1 or 2?

Reply:
good point !

Reply:
I made some change, default set "all" as default parameter, but just in case people develop new code and create  3 or 4 as parameter, I still do the parameter check